app/partials/disease-associations.html

```diff
@@ -22,10 +22,133 @@ <h3 view-title>{{search.total}} {{search.total===1 ? "target" : "targets"}} asso
             <div class="row" ng-if="search.total>0">
                 <div class="col-sm-3">
                     <cttv-facets></cttv-facets>
-                </div>
 
+                    <div>
```

Comment:
It would be nicer if we use a directive here and all this html is moved to that directive leaving only...
<filter-by-target-list></filter-by-target-list>

This will also help in making the controller cleaner